packages/salesforcedx-vscode-core/src/commands/forceApexTriggerCreate.ts

```diff
@@ -90,10 +86,7 @@ const fileNameGatherer = new SelectFileName();
 const filePathExistsChecker = new FilePathExistsChecker(APEX_TRIGGER_EXTENSION);
 
 export async function forceApexTriggerCreate(explorerDir?: any) {
```

Comment:
Do we need to keep the `explorerDir` parameter now that's not being used ? Same question applies to the other create commands being modified.

Reply:
I don't think so? I wasn't really sure what it was being used for in the first place...